pages/playlist/podcast/[id]/index.vue

```diff
@@ -1,3 +1,77 @@
+<script lang="ts" setup>
+const { t } = useI18n();
+toolbarTitleStore().setReactiveToolbarTitle(() => t("nav.podcast"));
+
+const { id } = useRoute<"playlist-podcast-id">().params;
+const collectionId = Number(id);
+
+const { data: podcast } = usePodcast({ id: collectionId });
+const { data: tracks, pending: tracksPending } = usePodcastTracks({
+  id: collectionId,
+});
+
+const trackGroups = computed(() => {
+  if (tracks.value) {
+    const sorted = tracks.value.sort((a, b) =>
+      a.publishedAt! > b.publishedAt! ? -1 : 1,
+    );
+
+    // split the tracks into groups of 7
```

Comment:
This just says the same as the code below. Useful would instead be a reasoning why you are doing this.

If you split in groups of 7, saying those are the episodes per week, you're assuming, that there's a new episode every day of a week (not only on working days) - additionally, you're assuming that the previous week ended 7 items before, which doesn't match if someone looks this up on a Wednesday.

I'd like to discuss this a bit before going ahead.

Reply:
Since we're talking about grouping the podcasts by the week they were published, I'd go for something like https://stackoverflow.com/a/6117889/517914